frontend/src/metabase/embedding-sdk/theme/embedding-color-palette.ts

```diff
@@ -110,4 +110,15 @@ export function setGlobalEmbeddingColors(
   Object.entries(combinedThemeColors).forEach(([key, value]) => {
     colors[key as ColorName] = value;
   });
+
+  /**
+   * (EMB-696)
+   * Reset colors set previously that's now not passed in `sdkColors`.
+   * Otherwise, previously modified colors will persist, and won't be reset to default values.
+   */
+  Object.keys(colors).forEach((key) => {
+    if (!combinedThemeColors[key as ColorName]) {
+      delete colors[key as ColorName];
+    }
+  });
```

Comment:
Oh wow! I was wondering why the colors never get cleared 🤦🏻‍♂️. Really great catch, I completely forgot that the colors also get set **globally** 🙏🏻